src/plone/base/interfaces/controlpanel.py

```diff
@@ -904,7 +904,7 @@ class INavigationSchema(Interface):
             "The content types that should be shown in the navigation and site map."
         ),
         required=False,
-        default=("Image", "File", "Link", "News Item", "Folder", "Document", "Event"),
+        default=("Link", "News Item", "Folder", "Document", "Event", "Collection"),
```

Comment:
@pbauer this might have consequences for plone.volto. If we show Collections in nav, we have to disable them in Volto.